code-review/README.md

```diff
@@ -0,0 +1,119 @@
+Code Review
+===========
+
+This is forked/stolen from ^W^W inspired by
+[Thoughtbot's code review guide](https://raw.githubusercontent.com/thoughtbot/guides/master/code-review/README.md)
+
+A guide for reviewing code and having your code reviewed.
+
+Why code reviews?
+-----------------
+
+Code is reviewed before being merged into `master`. The goal of review is to ensure that changes are delivering business value to our customers - not to perfect our code.
+
+When reviewing, consider whether a change:
+
+* Delivers business value. Understand why it is being made and why it is required.
+* Does what it says it does. Try to break it.
+* Can be maintained by the most junior member of our team.
+* Meets our standards and is well written.
+
+Don't let minor technical issues block shipping.
+
+A reviewer may give +1 on a review. Small fixes need +1, complex fixes and large features +2. A branch should have at least +1 before you hit the green button.
+
+Code Review Mindset
+-------------------
+
+It's important to separate our self-worth from the code we write so that we are always open to feedback that will help us improve.
+
+When reviewing code that changes user facing components it is important that reviews play the role of a User Advocate and consider how the change will impact our users, for example:
+* Is the copy written in such a way that it would be understand by someone who has learned English as their second language?
+* Is the change consistent with the user's mental model of our system?
+
+Everyone
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs
+  which you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands.
+    * *What do you think about naming this `:user_id`?*
+* Ask for clarification. 
+    * *I didn't understand. Can you clarify?*
+* Avoid selective ownership of code. 
+    * Avoid these words: "mine", "not mine", "yours"
+* Avoid using terms that could be seen as referring to personal traits.
+  ("dumb", "stupid"). Assume everyone is attractive, intelligent, and
+  well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. 
+    * *I'm not sure - let's look it up.*
+* Don't use hyperbole. 
+    * Avoid "always", "never", "endlessly", "nothing"
+* Don't use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor aren't you, don't force
+  them. If they are, use them with aplomb.
+* Talk in person if there are too many "I didn't understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
```

Comment:
Can I suggest also: _Make it clear whether your comment is **blocking** (you think it needs to be addressed before you grant a +1) or **non-blocking** (it's a suggestion or observation that the author can choose to implement or not). Without this distinction, it's not clear to the author what she needs to do to gain a +1._